process/README.md

```diff
@@ -129,9 +129,12 @@ The configuration for this automation is in
 _More on the Prow process in general
 [is available in the k8s docs](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process)._
 
-The Tekton community promotes company diversity as a best practice for pull request.
-This means that, where possible, one of the reviewers of a pull request and the author
-should be affiliated to different organizations.
+The Tekton community promotes company diversity as a best practice for pull requests
+to encourage a wide range of perspectives and feedback. This means that, where appropriate
+and possible, one of the reviewers of a pull request and the author should be affiliated to
+different organizations. If the changes are minor and reversible or have been already been
+discussed in a [TEP] with input and approval from different organizations, the pull request
+can be approved by the same organization.
```

Comment:
I don't think we should add caveats to this, since the caveats in themselves are ambiguous - e.g. what's a minor change, what was/was not already discussed, etc.

I'm okay with adjusting the language here to give maintainers more leeway though. Perhaps something like this?

```suggestion
The Tekton community promotes company diversity as a best practice for pull requests.
We encourage for large or significant changes (by maintainer discretion), that pull requests receive
feedback from multiple organizations.
```

Reply:
Agreed, maintainer discretion makes sense -- updating

Reply:
Updated